src/components/filters/Chip/Chip.css

```diff
@@ -0,0 +1,28 @@
+.g3-chip {
+  box-sizing: border-box;
+  font-size: 14px;
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  color: var(--g3-color__lightgray);
+  background-color: var(--g3-color__bg-cloud);
+  border-radius: 4px;
+}
+
+.g3-chip * {
+  box-sizing: inherit;
+}
+
+.g3-chip__text {
+  margin: 4px;
+  text-align: center;
+}
+
+.g3-chip__clear-btn {
```

Comment:
would it be better if we add `:hover` effect to this btn? i feel since the whole filter title is also a button it is a little bit confusing

Reply:
Yes, I see what you mean! I think hover states would be helpful, I'll see about adding them to the chip and the background.

Reply:
i would say only adding hover to the clear btn is enough, since the cursor will tell user the title and X are buttons and the hover seperates them from each other. But the final design is up to you 😃